MSAL/test/app/mac/MSALAcquireTokenViewController.m

```diff
@@ -467,4 +524,41 @@ - (MSALAccount *)selectedAccount
     return self.accounts[self.userPopup.indexOfSelectedItem-1];
 }
 
+- (MSALInteractiveTokenParameters *)tokenParamsWithSSOSeeding:(BOOL)isSSOSeedingCall
+{
+    MSALTestAppSettings *settings = [MSALTestAppSettings settings];
+    MSALInteractiveTokenParameters *parameters = [[MSALInteractiveTokenParameters alloc] initWithScopes:isSSOSeedingCall ? [MSALTestAppSettings getSSOSeedingScope] : [settings.scopes allObjects]
```

Comment:
nit: quite many conditionals in here? would it make sense to have two separate methods for readability?

e.g. tokenParams
tokenParamsForSSOSeedingCall

If there's same code between two, you can have a utility method to reuse code.

Reply:
Agree, I will re-structure this part a little bit for easier reading.